osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -69,41 +69,44 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
 
             double aimStrain = currVelocity; // Start strain with regular velocity.
 
-            if (Math.Max(osuCurrObj.StrainTime, osuLastObj.StrainTime) < 1.25 * Math.Min(osuCurrObj.StrainTime, osuLastObj.StrainTime)) // If rhythms are the same.
+            if (osuCurrObj.Angle != null && osuLastObj.Angle != null && osuLastLastObj.Angle != null)
```

Comment:
Noticed this previously but would be a good time to remove the `osuLastLastObj.Angle != null` requirement as we're no longer even using `osuLastLastObj.Angle` in here (or maybe not at all? maybe we can delete `osuLastLastObj` entirely?)